common/lib/xmodule/xmodule/js/src/sequence/display.coffee

```diff
@@ -153,7 +165,17 @@ class @Sequence
       new_position = $(event.currentTarget).data('element')
 
     if (1 <= new_position) and (new_position <= @num_contents)
-      Logger.log "seq_goto", old: @position, new: new_position, id: @id
+      is_bottom_nav = $(event.target).closest('nav[class="sequence-bottom"]').length > 0
+      if is_bottom_nav
+        widget_placement = 'bottom'
+      else
+        widget_placement = 'top'
+      Logger.log "edx.ui.lms.sequence.tab_selected",
```

Comment:
I think the comment is worth it. I should be able to very easily find this line of code using either the name or the event_type.